docs/configuration/settings.rst

```diff
@@ -59,7 +59,11 @@ Feature settings
 +-------------------------------------------------+--------------+---------------------------------------------------------------------------+
 | kinto.experimental_collection_schema_validation | ``False``    | *Experimental*: Allow definition of JSON schema at the collection level,  |
 |                                                 |              | in order to :ref:`validate submitted records <collection-json-schema>`.   |
-|                                                 |              | It is marked as experimental because the API might subjet to changes.     |
+|                                                 |              | It is marked as experimental because the API might be subject to changes. |
++-------------------------------------------------+--------------+---------------------------------------------------------------------------+
+| kinto.experimental_permissions_endpoint         | ``False``    | *Experimental*: Add a new permissions endpoint to let the user grab the   |
+|                                                 |              | list of ressources they can access to. Buckets/Collections/Records/Groups.|
```

Comment:
a new `/permissions` endpoint [...] list of objects (buckets, collections, groups, records) on which they have read or write permission.

?